Resources/Prototypes/Catalog/VendingMachines/Inventories/starkist.yml

```diff
@@ -8,6 +8,8 @@
     DrinkSolDryCan: 2
     DrinkLemonLimeCan: 2
     DrinkFourteenLokoCan: 2
+  contrabandInventory:
+    FoodMealSashimi: 2
```

Comment:
Probably too consistent considering is gives the very potent CarpoToxin

Reply:
Sashimi guarantees bunch of Opp for chemlab, I will defo be raiding kitchen every round if I am chem after it gets added... and restocks are only like 560$ each so thats very cheap roundstart mass cogni source as managament gets restocked too.

Reply:
To note, this is for the Star-Kisk Machine, it's not a consistent spawn on stations, and the Sashimi appears anywhere from 0-2.

The reasoning for the Sashimi is to deliberately have a fishy, but not Space Carp Fillet item to explain the Starkisk Can item description: "The taste of a star in liquid form. And, a bit of tuna...?" The soda only "tastes like orange soda", so I wanted this environmental bit to explain the fishiness, and it seemed like the funniest one to just find in a machine. 

When discussing this proposal on the SS14 discord, this has come up frequently, but no one has been able to explain to me how you get the Carpotoxin out of Sashimi short of extracting someone's blood after digesting it. Syringe and blender don't work. I've heard that the recycler and industrial reagent grinder can grind it up, but I can't get the recycler to break apart the sashimi on my private client and I've never once seen an industrial reagent grinder used in-game. Is there a verified way on how you actually accomplish this?

I'm also wondering if balance is actually affected given the current Salvage meta of doing Space Debris early, and frequently. An average roll debris is ~3 carp, which is 6 fillet if butchered on the spot, which is becoming a more common practice in my gameplay. 3 debris, 18 fillet, and it can easily high-roll even better than this. I'm not sure what a potential 4 fillet worth of Carpotoxin from a high roll sashimi spawn in a Starkisk machine is going to do here. That is to say, Carpotoxin doesn't currently feel rare when I play Salvager, or as Chemist, especially if you just ask Salvage to take the few seconds to farm it. It feels like the effort to farm a Starkisk machine for fillets, assuming it's currently possible, is more work than Salvage bothering to butcher Space Carp while they work.

In terms of substitutions, we can reduce yield by using the fillet-o-carp burger which has half the fillet yield, or just throw a space carp plushie in and find something else to go into the Salvage machine.

Reply:
> When discussing this proposal on the SS14 discord, this has come up frequently, but no one has been able to explain to me how you get the Carpotoxin out of Sashimi short of extracting someone's blood after digesting it.

Just drain it in chemlab/morgue or eat and vomit if you extremely concerned about wasting any unit of it and have enough ipecac.

Reply:
Sorry, what do you mean by drain it in the chemlab or morgue? 

And yeah, vomiting makes way more sense to me as a form of improvised extraction. I think that'd be an interesting mechanic, but yeah, I could see that being an issue.

Reply:
Oh crap, that actually works. I had no idea this was a thing with cooked foods: 
![image](https://github.com/user-attachments/assets/b63f022c-fa8b-4b64-875e-7195c47caac8)

To note, the Fillet-O-Carp burger doesn't contain Carpotoxin. I'll make the substitution, good catch:
![image](https://github.com/user-attachments/assets/df52813a-8005-4e52-aefc-4410a506e76b)

Reply:
Fillet-O-Carp is now in the machine instead of Sashimi.

Reply:
> Oh crap, that actually works. I had no idea this was a thing with cooked foods: ![image](https://private-user-images.githubusercontent.com/182301147/378466363-b63f022c-fa8b-4b64-875e-7195c47caac8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1NDk4MzQsIm5iZiI6MTczMDU0OTUzNCwicGF0aCI6Ii8xODIzMDExNDcvMzc4NDY2MzYzLWI2M2YwMjJjLWZhOGItNGI2NC04NzVlLTcxOTVjNDdjYWFjOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMlQxMjEyMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOWJjNWY2ZDVkYjU5ZmYyOWE2MzU0NTEzYWNlZGUxNWVhNWY3NTVmMTA5MzA3OGM5NDFjOTJjNzE3YmIzOGE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Niz6hWMb1iIqiDyIOtJTHa66_kmk3tOYj_zY87DehT0)
> 
> To note, the Fillet-O-Carp burger doesn't contain Carpotoxin. I'll make the substitution, good catch: ![image](https://private-user-images.githubusercontent.com/182301147/378467199-df52813a-8005-4e52-aefc-4410a506e76b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1NDk4MzQsIm5iZiI6MTczMDU0OTUzNCwicGF0aCI6Ii8xODIzMDExNDcvMzc4NDY3MTk5LWRmNTI4MTNhLTgwMDUtNGU1Mi1hZWZjLTQ0MTBhNTA2ZTc2Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMlQxMjEyMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NmU4ODUzOTc5YTJhMjJhM2NiMmMxYWJkMWQyMWU4NDNjOGRlN2RiMjk4NWM2OWYxMDM4YmMzNjBkYzhmYWRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.4j6-Qw0jYDTrw2ig8KZGN375yS0tCgfvKBwyrl_NJKw)

So Lilten made new [video ](https://www.youtube.com/watch?v=tRWHrX4aIDk)and I realized the bug have evolved as you were not able to to drain food directly into chemmaster(it would only play sound)... Now that I think about it maybe jugs were not able to always drain too? Perhaps thats when it changed. I was using floor drains for no reason and dilluting plant chems to minimize the loss. x)

Reply:
Wow, I was not ready for that video, lol. Yeah, it's going to have implications, and I've already given Sashimi to Chemistry to drain out its Carpotoxin in one of my rounds.

I wonder how long the bug is going to last. Otherwise, now that I know it's _possible_, I'm going to avoid the possibility of it being easy chems like an Arnolds Pizza is.